CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## v1.15.2 (2021-08-11)
+
+### Bug Fixes
+
+* TestCafe fails to emulate the `tab` action if a page contains a cross-domain iframe ([#6405](https://github.com/DevExpress/testcafe/issues/6405)).
+* TestCafe ignores the quarantine mode options if you specify the configuration file ([#6420](https://github.com/DevExpress/testcafe/issues/6420)).
```

Comment:
What does `specify the configuration file` mean? Do you mean change it or add it?